qa/rpc-tests/wallet.py

```diff
@@ -72,6 +72,36 @@ def run_test (self):
         node0utxos = self.nodes[0].listunspent(1)
         assert_equal(len(node0utxos), 3)
 
+        # Check 'generated' field of listunspent
+        #
+        # Node 0: has one coinbase utxo and two regular utxos
+        generatedCounter = 0
+        for utxo in node0utxos:
+            if utxo["generated"] is True:
+                generatedCounter = generatedCounter + 1
+        assert(generatedCounter == 1)
```

Comment:
You can write this as ``assert_equal(sum(int(uxto["generated"] is True) for uxto in node0utxos), 1)``, and similarly below.